tools/serverpod_cli/lib/src/test_util/builders/model_source_builder.dart

```diff
@@ -11,12 +13,18 @@ class ModelSourceBuilder {
     fields:
       name: String
     ''');
-    yamlSourceUri = Uri(path: 'lib/src/model/example.yaml');
+    yamlSourcePath = 'lib/src/model/example.yaml';
```

Comment:
This changes from URI (that is maybe handled from the OS level) to just this string. Could that be an issue?

Reply:
Orr that should only be an issue for tests using this, and I don't think the tests we are looking at use this.

Reply:
Either way we should probably build the path using the Path lib to not have issues on windows in the future.